samples/ApptentiveSample/Properties/AndroidManifest.xml

```diff
@@ -1,10 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.apptentive.xamarin.ApptentiveAndroidSample">
-	<uses-sdk android:minSdkVersion="28" />
-	<application android:allowBackup="true" android:icon="@mipmap/icon" android:label="@string/app_name">
-		<activity android:name=".UserDataActivity" android:label="User Data" android:theme="@style/ApptentiveTheme.Base.Versioned" />
-		<activity android:name=".CustomDataActivity" android:label="Custom Data" android:theme="@style/ApptentiveTheme.Base.Versioned" />
-		<activity android:name=".LoginActivity" android:label="Login/Logout" android:theme="@style/ApptentiveTheme.Base.Versioned" />
+	<uses-sdk android:minSdkVersion="30" android:targetSdkVersion="33" />
```

Comment:
Did the app force you to keep the minSdk as "30"?

Reply:
Also curious about this. We can make the minSDK here to 21. 30 is pretty high.

Reply:
I changed it to 21.